cdn/sign_url.rb

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def signed_url url:, key_name:, key_file:, expiration:
+  # [START signed_url]
+  # url        = "URL to content served by Google Cloud CDN"
+  # key_name   = "Signing key name"
+  # key_file   = "Signing key file name with a urlsafe base64 encoded key"
+  # expiration = "Expiration time for signed URL formatted as a Unix Timestamp"
```

Comment:
doesn't ruby have a datetime type? it'd be preferable to use the language-idiomatic format instead of having everyone convert to unix time.

Reply:
It does, I can update this to use a Time object instead.